### PR TITLE
Fixes #4984 Fix problem with RUCSS and ConvertPlug

### LINKED ISSUE #4984: Remove Unused CSS compatibility with ConvertPlug dynamic .content-
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.11.0.5`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
ConvertPlug adds popups with dynamic IDs with their respective style added inline just below them. 

```
<div class="content-626b819000a4c" ... ></div>
<style type="text/css" class="cp-form-css">
  .content-626b819000a4c ... {
  }
</style>
```

When the cache is regenerated, the above class will change. 

Excluding `cp-form-css` using `rocket_rucss_inline_atts_exclusions` fixes the issue. 

We can permanently exclude it from https://github.com/wp-media/wp-rocket/blob/9f68883a4c3468359bcc3fd7b9779b8bf7f72380/inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php#L60-L69 to prevent the issue from happening. 

**To Reproduce**
1. Use ConvertPlug
2. Add a popup opt-in form on the page
3. Generate Used CSS
4. Clear the cache
5. See error

**Expected behavior**
ConvertPlugin style should be retained even after clearing the cache when Remove Unused CSS is used. 

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1863092210/339787

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

### PR DESCRIPTION
**Pull request by CrochetFeve0251**
## Description

Fix a conflit between  RUCSS and ConvertPlug.
This has been fixed by excluding the CSS in question in a thirdparty class.

Fixes #4984

## Type of change

- [ ] Enhancement (non-breaking change which improves an existing functionality)

## Is the solution different from the one proposed during the grooming?

No.

## How Has This Been Tested?

- [ ] Automated Tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/Integration/inc/ThirdParty/Plugins/ConvertPlug/excludedFromRucss.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * @covers  \WP_Rocket\ThirdParty\Plugins\Ecommerce\WooCommerceSubscriber::maybe_revert_uid_for_nonce_actions
```
Comment (remyperona) [inline tests/Integration/inc/ThirdParty/Plugins/ConvertPlug/excludedFromRucss.php]: This needs to refer to the correct method

File: tests/Unit/inc/ThirdParty/Plugins/ConvertPlug/excludedFromRucss.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+namespace WP_Rocket\Tests\Unit\inc\ThirdParty\Plugins\ConvertPlug;
+
+use WP_Rocket\Tests\Unit\TestCase;
+use WP_Rocket\ThirdParty\Plugins\ConvertPlug;
+
```
Comment (remyperona) [inline tests/Unit/inc/ThirdParty/Plugins/ConvertPlug/excludedFromRucss.php:6]: missing the cover and group annotations

**Review by remyperona — CHANGES_REQUESTED** [review]

File: inc/ThirdParty/Plugins/ConvertPlug.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	}
+
+	/**
+	 * Exclude css from RUCSS.
+	 *
+	 * @param array $excluded excluded css.
+	 * @return array
+	 */
+	public function excluded_from_rucss( $excluded ) {
+		$excluded[] = '.content-';
```
Comment (remyperona) [inline inc/ThirdParty/Plugins/ConvertPlug.php]: Shouldn't the value be `cp-form-css` instead?

**Review by remyperona — CHANGES_REQUESTED** [review]

**Review by remyperona — APPROVED** [review]

**Review by piotrbak — APPROVED** [review]

**Comment by piotrbak** [comment]
@Tabrisrp This one too 🙏

**Pull request merged**